tests/Test.MSAL.NET.Unit/CacheTests/RefreshTokenCacheKeyTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using Microsoft.Identity.Client.Internal.Cache;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Test.MSAL.NET.Unit.CacheTests
+{
+    [TestClass]
+    public class RefreshTokenCacheKeyTests
+    {
+        [TestMethod]
+        [TestCategory("RefreshTokenCacheKeyTests")]
+        public void ConstructorTest()
```

Comment:
Related to an earlier comment: unless _RefreshTokenCacheKey_ objects are directly compared against each other or used as keys in dictionary then there's no point in overriding _Equals_ and _GetHashCode_, so these tests wouldn't be needed either.